numba/core/utils.py

```diff
@@ -495,6 +493,10 @@ def unified_function_type(numba_types):
     If any of the Numba function types is a Dispatcher type, the
     unified function type will be UndefinedFunctionType instance.
     """
+    from numba.core.errors import NumbaExperimentalFeatureWarning
```

Comment:
Move import to the top? Or is there circular import problems.

Reply:
it's a circular import problem.